arduino-test/build-sketches.js

```diff
@@ -1,99 +1,95 @@
-'use strict';
+"use strict";
 
-const fs = require('fs');
-const child_process = require('child_process');
+import { mkdirSync, writeFileSync } from "fs";
+import { execSync } from "child_process";
 
-const SketchTemplater = require('@sensebox/sketch-templater');
+import SketchTemplater from "@sensebox/sketch-templater";
 
-const sketchTemplater = new SketchTemplater('test.ingress.domain');
+const sketchTemplater = new SketchTemplater("test.ingress.domain");
 
 const boxStub = function (model) {
```

Comment:
The following sensors/phenoma are missing:
- BME680 (Temperature, Humidity, IAQ)
- Windspeed (Windspeed)
- Soundsensor 
- SCD30 CO2 Sensor (CO2, Temperature, Humidity)
- Soilmoisture and Temperature Sensor (Soilmoisture, Temperature)